### PR TITLE
Refactoring circle page renderer

### PR DESCRIPTION
**Pull request by myroot**
### Description of Change ###
Refactoring CirclePageRenderer
 * Remove _bgColorObject : not need, background color can be supported by box
 * Lazy creation for _bgImageObject
 * Remove Dictionary for _toolbarItemMap : replace to FormsMoreOptionItem

### Bugs Fixed ###
 None

### API Changes ###
None

### Behavioral Changes ###
None

### CODE REVIEW

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/CirclePageRenderer.cs [hunk 1]
```diff
@@ -122,340 +105,391 @@ protected override void OnElementReady()
...
+            Children.Remove(_actionButton);
+            _actionButton.Unrealize();
+            _actionButton = null;
+        }
 
-            }
+        void InitializeBgImageObject()
+        {
+            _bgImageObject = new EvasImage(XForms.NativeParent);
+            _bgImageObject.IsFilled = true;
```
Comment (rookiejava) [inline src/Tizen.Wearable.CircularUI.Forms.Renderer/CirclePageRenderer.cs:216]:
```suggestion
            _bgImageObject = new EvasImage(XForms.NativeParent)
            {
                IsFilled = true;
            };
```

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/CirclePageRenderer.cs [hunk 2]
```diff
@@ -122,340 +105,391 @@ protected override void OnElementReady()
...
-                _moreOption.StackAbove(prev);
+                if (_bgImageObject != null)
+                {
+                    DeinitializeBgImageObject();
+                }
             }
         }
 
-        void UpdateBackground()
+        void UpdateActionButton(bool init)
```
Comment (rookiejava) [inline src/Tizen.Wearable.CircularUI.Forms.Renderer/CirclePageRenderer.cs:268]: Use `init` to skip unnecessary code at initial time or just make `UpdateActionButton()` if you don't need.

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/CirclePageRenderer.cs [hunk 3]
```diff
@@ -122,340 +105,391 @@ protected override void OnElementReady()
...
             }
         }
 
-        void OnActionButtonItemChanged(object sender, PropertyChangedEventArgs e)
+        void UpdateActionButtonVisible(bool visible)
         {
             if (_actionButton == null)
             {
                 return;
             }
```
Comment (rookiejava) [inline src/Tizen.Wearable.CircularUI.Forms.Renderer/CirclePageRenderer.cs:329]: we can skip these lines

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/CirclePageRenderer.cs (hunk 2 above)
Comment (myroot) [reply src/Tizen.Wearable.CircularUI.Forms.Renderer/CirclePageRenderer.cs]: Done

File: src/Tizen.Wearable.CircularUI.Forms.Renderer/CirclePageRenderer.cs (hunk 1 above)
Comment (myroot) [reply src/Tizen.Wearable.CircularUI.Forms.Renderer/CirclePageRenderer.cs]: Done

**Comment by myroot** [comment]
I decide to refactoring with PageRenderer on this(https://github.com/Samsung/Tizen.CircularUI/pull/291) PR

**Pull request closed without merging**